frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.tsx

```diff
@@ -54,6 +58,8 @@ interface StateProps {
   actionId?: number;
   question?: Question;
   metadata: Metadata;
+  isAdmin: any;
```

Comment:
`boolean`?

Reply:
dang. Forgot to fix this. I put any there so I know I have to come back to fix it later, apparently forgot to annotate it with todo comment so I haven't fixed it in the end. Thanks for pointing this out.